diff -uNr kvirc/acinclude.m4~ kvirc/acinclude.m4
--- kvirc/acinclude.m4~	2004-02-11 15:08:53.000000000 +0100
+++ kvirc/acinclude.m4	2004-02-11 15:26:24.741986624 +0100
@@ -1937,10 +1937,10 @@
 	helpdir="\${globalkvircdir}/help/en"
 	AC_SUBST(helpdir)
 
-	pluglibdir="\${globalkvircdir}/modules"
+	pluglibdir="\${exec_prefix}/lib/kvirc/$VERSION/modules"
 	AC_SUBST(pluglibdir)
 
-	modulelibdir="\${globalkvircdir}/modules"
+	modulelibdir="\${exec_prefix}/lib/kvirc/$VERSION/modules"
 	AC_SUBST(modulelibdir)
 
 	defscriptdir="\${globalkvircdir}/defscript"
@@ -2165,15 +2165,15 @@
 	AC_SUBST(kdeservicesdir)
 
 	if test "$SS_KDE_REQUESTED" = "yes"; then
-		iconapps16datadir="$SS_KDE_GENERAL_DIR/share/icons/hicolor/16x16/apps"
-		iconapps32datadir="$SS_KDE_GENERAL_DIR/share/icons/hicolor/32x32/apps"
-		iconapps48datadir="$SS_KDE_GENERAL_DIR/share/icons/hicolor/48x48/apps"
-		iconapps64datadir="$SS_KDE_GENERAL_DIR/share/icons/hicolor/64x64/apps"
-		iconmime16datadir="$SS_KDE_GENERAL_DIR/share/icons/hicolor/16x16/mimetypes"
-		iconmime32datadir="$SS_KDE_GENERAL_DIR/share/icons/hicolor/32x32/mimetypes"
-		iconmime48datadir="$SS_KDE_GENERAL_DIR/share/icons/hicolor/48x48/mimetypes"
-		iconmime64datadir="$SS_KDE_GENERAL_DIR/share/icons/hicolor/64x64/mimetypes"
-		applnkdir="$SS_KDE_GENERAL_DIR/share/applnk/Internet"
+		iconapps16datadir="${kde_icondir}/hicolor/16x16/apps"
+		iconapps32datadir="${kde_icondir}/hicolor/32x32/apps"
+		iconapps48datadir="${kde_icondir}/hicolor/48x48/apps"
+		iconapps64datadir="${kde_icondir}/hicolor/64x64/apps"
+		iconmime16datadir="${kde_icondir}/hicolor/16x16/mimetypes"
+		iconmime32datadir="${kde_icondir}/hicolor/32x32/mimetypes"
+		iconmime48datadir="${kde_icondir}/hicolor/48x48/mimetypes"
+		iconmime64datadir="${kde_icondir}/hicolor/64x64/mimetypes"
+		applnkdir="${kde_appsdir}/Internet"
 		mimelnkdir="$SS_KDE_GENERAL_DIR/share/mimelnk/text"
 	else
 		iconapps16datadir="\${exec_prefix}/share/kvirc/$VERSION/icons/16x16"
diff -uNr kvirc/src/kvirc/kernel/kvi_app_fs.cpp~ kvirc/src/kvirc/kernel/kvi_app_fs.cpp
--- kvirc/src/kvirc/kernel/kvi_app_fs.cpp~	2003-11-20 02:53:38.000000000 +0100
+++ kvirc/src/kvirc/kernel/kvi_app_fs.cpp	2004-02-11 15:26:24.744986168 +0100
@@ -66,7 +66,7 @@
 		case None          :                                                                           break;
 		case Pics          : szData.append("pics");                                                    break;
 		case Modules       :
-		case Plugins       : szData.append("modules");                                                 break;
+		case Plugins       : szData="/usr/lib/kvirc/3.0.0-beta3/modules";		               break;
 		case ConfigPlugins : szData.append(KviStr::Format,"config%smodules",KVI_PATH_SEPARATOR);       break;
 		case ConfigScripts : szData.append(KviStr::Format,"config%sscripts",KVI_PATH_SEPARATOR);       break;
 		case Help          :
			#endif
--- kvirc/src/kvirc/kernel/kvi_app_setup.cpp.niedakh	2003-09-12 04:23:11.000000000 +0200
+++ kvirc/src/kvirc/kernel/kvi_app_setup.cpp	2004-02-17 14:22:55.707173504 +0100
@@ -70,8 +70,8 @@
 bool KviApp::checkGlobalKvircDirectory(const char *dir)
 {
 	//First check if the help subdir exists
-	KviStr szDir2(KviStr::Format,"%s%smodules",dir,KVI_PATH_SEPARATOR);
-	if(!kvi_directoryExists(szDir2.ptr()))return false;
+//	KviStr szDir2(KviStr::Format,"%s%smodules",dir,KVI_PATH_SEPARATOR);
+//	if(!kvi_directoryExists(szDir2.ptr()))return false;
 	//Then check if the pics subdir exists
 	KviStr szDir(KviStr::Format,"%s%spics",dir,KVI_PATH_SEPARATOR);
 	if(!kvi_directoryExists(szDir.ptr()))return false;
@@ -124,6 +124,7 @@
 {
 	// Look for the installation directory
 	// with the image and help files inside
+#define	GLOBAL_KVIRC_DIR "/usr/share/kvirc/"
 #ifdef GLOBAL_KVIRC_DIR
 	m_szGlobalKvircDir = GLOBAL_KVIRC_DIR;
 	kvi_adjustFilePath(m_szGlobalKvircDir);
@@ -307,10 +308,10 @@
 	// First find the global (readable) one...
 	findGlobalKvircDirectory();
 	// Init locale
-	KviStr szLocalePath = m_szGlobalKvircDir;
-	szLocalePath.append(KviStr::Format,"%slocale%s",KVI_PATH_SEPARATOR,KVI_PATH_SEPARATOR);
+//	KviStr szLocalePath = m_szGlobalKvircDir;
+//	szLocalePath.append(KviStr::Format,"%slocale%s",KVI_PATH_SEPARATOR,KVI_PATH_SEPARATOR);
 
-	KviLocale::init(this,szLocalePath.ptr());
+	KviLocale::init(this,NULL);
 
 	//__debug_1arg("Global Kvirc directory is %s",m_szGlobalKvircDir.ptr());
 	// Now look for the local (writable) one
--- kvirc/src/kvilib/system/kvi_locale.cpp.niedakh	2003-12-24 12:51:11.000000000 +0100
+++ kvirc/src/kvilib/system/kvi_locale.cpp	2004-02-17 19:21:48.497974920 +0100
@@ -429,7 +429,7 @@
 		KviStr szLocDir = szLocaleDir;
 		szLocDir.ensureLastCharIs(KVI_PATH_SEPARATOR_CHAR);
 
-		szBuffer.sprintf("%s%s_%s.mo",szLocDir.ptr(),name,szLocale.ptr());
+		szBuffer.sprintf("%s%s.mo",szLocDir.ptr(),name);
 		//debug("Lookup %s",szBuffer.ptr());
 		if(kvi_fileExists(szBuffer.ptr()))return true;
 
@@ -469,7 +469,7 @@
 	}
 
 
-	void init(QApplication * app,const char * localeDir)
+	void init(QApplication * app,const char * oldlocaleDir)
 	{
 		// first of all try to find out the current locale
 		g_szLang = kvi_getenv("KVIRC_LANG");
@@ -761,6 +761,10 @@
 		// the Qt docs explicitly state that we shouldn't delete
 		// the codecs by ourselves...
 		g_pSmartCodecDict->setAutoDelete(false);
+		
+		#include <qstring.h>
+ 		QString nlocaleDir = QString("/usr/share/locale/") + QString (g_szLang) + QString("/LC_MESSAGES/");
+ 		const char * localeDir = nlocaleDir.latin1();
 
 		if(g_szLang.hasData())
 		{
