diff -uNr kvirc/acinclude.m4~ kvirc/acinclude.m4
--- kvirc/acinclude.m4~	2004-02-11 15:08:53.000000000 +0100
+++ kvirc/acinclude.m4	2004-02-11 15:26:24.741986624 +0100
@@ -1937,10 +1937,10 @@
 	helpdir="\${globalkvircdir}/help/en"
 	AC_SUBST(helpdir)
 
-	pluglibdir="\${globalkvircdir}/modules"
+	pluglibdir="\${exec_prefix}/lib/kvirc/$VERSION/modules"
 	AC_SUBST(pluglibdir)
 
-	modulelibdir="\${globalkvircdir}/modules"
+	modulelibdir="\${exec_prefix}/lib/kvirc/$VERSION/modules"
 	AC_SUBST(modulelibdir)
 
 	defscriptdir="\${globalkvircdir}/defscript"
@@ -1963,11 +1963,11 @@
 	AC_SUBST(kdeservicesdir)
 
 	if test "$SS_KDE_REQUESTED" = "yes"; then
-		iconapps32datadir="$SS_KDE_GENERAL_DIR/share/icons/hicolor/32x32/apps"
-		iconapps48datadir="$SS_KDE_GENERAL_DIR/share/icons/hicolor/48x48/apps"
-		iconmime32datadir="$SS_KDE_GENERAL_DIR/share/icons/hicolor/32x32/mimetypes"
-		iconmime48datadir="$SS_KDE_GENERAL_DIR/share/icons/hicolor/48x48/mimetypes"
-		applnkdir="$SS_KDE_GENERAL_DIR/share/applnk/Internet"
+		iconapps32datadir="${kde_icondir}/hicolor/32x32/apps"
+		iconapps48datadir="${kde_icondir}/hicolor/48x48/apps"
+		iconmime32datadir="${kde_icondir}/hicolor/32x32/mimetypes"
+		iconmime48datadir="${kde_icondir}/hicolor/48x48/mimetypes"
+		applnkdir="${kde_appsdir}/Internet"
 		mimelnkdir="$SS_KDE_GENERAL_DIR/share/mimelnk/text"
 	else
 		iconapps32datadir="\${exec_prefix}/share/kvirc/$VERSION/icons/32x32"
diff -uNr kvirc/src/kvirc/kernel/kvi_app_fs.cpp~ kvirc/src/kvirc/kernel/kvi_app_fs.cpp
--- kvirc/src/kvirc/kernel/kvi_app_fs.cpp~	2003-11-20 02:53:38.000000000 +0100
+++ kvirc/src/kvirc/kernel/kvi_app_fs.cpp	2004-02-11 15:26:24.744986168 +0100
@@ -66,7 +66,7 @@
 		case None          :                                                                           break;
 		case Pics          : szData.append("pics");                                                    break;
 		case Modules       :
-		case Plugins       : szData.append("modules");                                                 break;
+		case Plugins       : szData="/usr/lib/kvirc/" VERSION "/modules";		               break;
 		case ConfigPlugins : szData.append(KviStr::Format,"config%smodules",KVI_PATH_SEPARATOR);       break;
 		case ConfigScripts : szData.append(KviStr::Format,"config%sscripts",KVI_PATH_SEPARATOR);       break;
 		case Help          :
			#endif
diff -uNr kvirc/src/kvilib/system/kvi_locale.cpp.niedakh kvirc/src/kvilib/system/kvi_locale.cpp
--- kvirc/src/kvilib/system/kvi_locale.cpp.niedakh	2003-12-24 12:51:11.000000000 +0100
+++ kvirc/src/kvilib/system/kvi_locale.cpp	2004-02-12 18:07:35.513485600 +0100
@@ -429,7 +429,7 @@
 		KviStr szLocDir = szLocaleDir;
 		szLocDir.ensureLastCharIs(KVI_PATH_SEPARATOR_CHAR);
 
-		szBuffer.sprintf("%s%s_%s.mo",szLocDir.ptr(),name,szLocale.ptr());
+		szBuffer.sprintf("%s%s.mo",szLocDir.ptr(),name);
 		//debug("Lookup %s",szBuffer.ptr());
 		if(kvi_fileExists(szBuffer.ptr()))return true;
 
@@ -469,7 +469,7 @@
 	}
 
 
-	void init(QApplication * app,const char * localeDir)
+	void init(QApplication * app,const char * oldlocaleDir)
 	{
 		// first of all try to find out the current locale
 		g_szLang = kvi_getenv("KVIRC_LANG");
@@ -484,7 +484,9 @@
 		// the catalogue dict
 		g_pCatalogueDict = new QAsciiDict<KviMessageCatalogue>;
 		g_pCatalogueDict->setAutoDelete(true);
-
+#include <string.h>
+		char * localeDir="/usr/share/locale";
+		strcat (localeDir, g_szLang);
 		if(g_szLang.hasData())
 		{
 			KviStr szBuffer;
diff -uNr kvirc/src/kvirc/kernel/kvi_app_setup.cpp.niedakh kvirc/src/kvirc/kernel/kvi_app_setup.cpp
--- kvirc/src/kvirc/kernel/kvi_app_setup.cpp.niedakh	2003-09-12 04:23:11.000000000 +0200
+++ kvirc/src/kvirc/kernel/kvi_app_setup.cpp	2004-02-12 18:14:05.148252112 +0100
@ -124,6 +124,7 @@
 {
 	// Look for the installation directory
 	// with the image and help files inside
+#define	GLOBAL_KVIRC_DIR "/usr/share/kvirc/"
 #ifdef GLOBAL_KVIRC_DIR
 	m_szGlobalKvircDir = GLOBAL_KVIRC_DIR;
 	kvi_adjustFilePath(m_szGlobalKvircDir);
@ -307,10 +308,10 @@
 	// First find the global (readable) one...
 	findGlobalKvircDirectory();
 	// Init locale
-	KviStr szLocalePath = m_szGlobalKvircDir;
-	szLocalePath.append(KviStr::Format,"%slocale%s",KVI_PATH_SEPARATOR,KVI_PATH_SEPARATOR);
+//	KviStr szLocalePath = m_szGlobalKvircDir;
+//	szLocalePath.append(KviStr::Format,"%slocale%s",KVI_PATH_SEPARATOR,KVI_PATH_SEPARATOR);
 
-	KviLocale::init(this,szLocalePath.ptr());
+	KviLocale::init(this,NULL);
 
 	//__debug_1arg("Global Kvirc directory is %s",m_szGlobalKvircDir.ptr());
 	// Now look for the local (writable) one
