--- src/modules/options/optw_irc.cpp.orig	2004-02-09 14:27:35.000000000 +0100
+++ src/modules/options/optw_irc.cpp	2004-02-11 16:40:27.137639832 +0100
@@ -63,7 +63,7 @@
     
 	KviBoolSelector *b4 = addBoolSelector(0,4,0,4,__tr("Auto join channel on INVITE"),KviOption_boolAutoJoinOnInvite);
 #ifdef COMPILE_INFO_TIPS
-	QToolTip::add(b4
+	QToolTip::add(b4,
 		__c2q(__tr("<center>This option will cause KVIrc to automatically<br>" \
 			"join a channel when an INVITE message for that channel is received.<br>" \
 			"Note that this may help spammers in bothering you :)</center>")));
